src/cloudant/document.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, database, document_id=None):
         self._document_id = document_id
         if self._document_id is not None:
             self['_id'] = self._document_id
-        self._encoder = self._client.encoder
+        self.encoder = self._client.encoder
```

Comment:
All instances of `self.encoder` should be set back to `self._encoder` in the Document.  We talked about this.